en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
+- silverstripe/session-manager [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
+- silverstripe/recipe-ccl [![CI](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml)
+- silverstripe/recipe-content-blocks [![CI](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml)
+- silverstripe/recipe-core [![CI](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml)
+- silverstripe/staticpublishqueue [![CI](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml)
+
+#### Other issues
+
+`tractorcow/silverstripe-fluent` is temporarily being release under `silverstripe/silverstripe-fluent`.
+
+## Upgrading your third party module
+
+One of the primary purposes of the beta testing phase is to give third party maintainers and developers an opportunity to upgrade their modules to be compatible with Silverstripe CMS 5.
+
+### Planning your upgrade
+
+You should decide if you intend to do a new major release of your module to add support for Silverstripe CMS 5.
+
+Most of key Silverstripe CMS APIs have not change between Silverstripe CMS 4 and Silverstripe CMS 5. Take time to read this changelog to identify changes that might be problematic for your project.
+
+Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach. But you may wish to consider it to minimise your maintenance burden.
+
+If you decide that you want to maintain distinct Silverstripe CMS 4 and Silverstripe CMS 5 versions of your module, then you should fork your current development branch.
+
+[info]
+If the name of your branch is a number, composer will allow it to be installed as an unstable tag. e.g.: If your module has a `7` branch, composer will alias the `HEAD` of that branch as `7.x-dev`. This will satisfy a constraint like `^7`.
+[/info]
+
+### Stop using deprecated code
+
+A good way to identify which APIs need to be updated is to [enable deprecation warnings](/Upgrading/Deprecations) and run your module against the latest Silverstripe CMS 4 release. This will let you remove any references to deprecated APIs in the Silverstripe CMS 4 version of your module.
```

Comment:
```suggestion
A good way to identify which APIs need to be updated is to [enable deprecation warnings](/Upgrading/Deprecations) and run your module in a fresh project using the latest Silverstripe CMS 4 release. This will let you remove any references to deprecated APIs in the Silverstripe CMS 4 version of your module.
```